### ISSUE TITLE
temp * temp * 2.0 * alpha 的意思

### ISSUE THREAD

**Issue by Duducoco** [mepeichun/Efficient-Neural-Network-Bilibili#3]
个人理解的是:
<math display="block" xmlns="http://www.w3.org/1998/Math/MathML"><mrow><mi mathvariant="italic">l</mi><mi mathvariant="italic">o</mi><mi mathvariant="italic">s</mi><mi mathvariant="italic">s</mi><mo>&#x3D;</mo><mi mathvariant="italic">a</mi><mi mathvariant="italic">p</mi><mi mathvariant="italic">h</mi><mi mathvariant="italic">a</mi><mo>&#x2A;</mo><mrow><mi mathvariant="italic">L</mi><mi mathvariant="italic">O</mi><mi mathvariant="italic">S</mi><mi mathvariant="italic">S</mi><msub><mrow></mrow><mrow><mi mathvariant="italic">s</mi><mi mathvariant="italic">o</mi><mi mathvariant="italic">f</mi><mi mathvariant="italic">t</mi></mrow></msub><mo>&#x2B;</mo></mrow><mo>&#x28;</mo><mn>1</mn><mo>&#x2D;</mo><mi mathvariant="italic">a</mi><mi mathvariant="italic">l</mi><mi mathvariant="italic">p</mi><mi mathvariant="italic">h</mi><mi mathvariant="italic">a</mi><mo>&#x29;</mo><mo>&#x2A;</mo><mi mathvariant="italic">L</mi><mi mathvariant="italic">O</mi><mi mathvariant="italic">S</mi><mi mathvariant="italic">S</mi><msub><mrow></mrow><mrow><mi mathvariant="italic">h</mi><mi mathvariant="italic">a</mi><mi mathvariant="italic">r</mi><mi mathvariant="italic">d</mi></mrow></msub></mrow></math>
不知道temp\*temp*2.0是什么意思

**Comment by mepeichun** [comment]
你的理解是正确的。其实不需要temp*temp*2.0, 我一开始参考了别人的实现，后边看了keras的官方实现，其实直接用 loss = self.alpha * student_loss + (1 - self.alpha) * distillation_loss就可以
https://keras.io/examples/vision/knowledge_distillation/

感谢你的纠正。我稍后会修改代码。
